org.civicrm.bootstrapcivihr/scss/mixins/_spinner.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Spinner mixin
+ * $speed Speed to rorate the spinner in seconds
+ * $height Height of the spinner
+ * $height Width of the spinner
+ */
+@mixin spinner($speed: 1s, $height: 32px, $width: 32px) {
```

Comment:
you are still able to define different width and height, which is something we *do not* want to happen.

Reply:
Correct! lets pass a param called `$size: 32px`  whose default size will be `32px`.